### ISSUE TITLE
Alternative text of the Confederation logo

### ISSUE THREAD

**Issue by desobryc** [swiss/styleguide#248]
![Uploading Nicht-text inhalt.PNG . . .]()
Texts about the logo https://www.webguidelines.admin.ch/design_principles_-_header_and_footer.html and https://www.webguidelines.admin.ch/base_layout_-_general.html#a-logo
should be completed:
The alternative text of the link should be "back to home", the link provided should be linking to the home page of the considered website (www.eda.admin.ch, www.bag.admin.ch) and not to admin.ch.
